### ISSUE TITLE
Is OGCS compatible with Outlook 2019?

### ISSUE THREAD

**Issue by schwaben-github** [phw198/OutlookGoogleCalendarSync#1208]
**Are you already using OGCS?**
Yes, I am, but since it isn't starting right now, I'm not able to answer this.
I suppose, it's the latest version.

**Describe the question you have**
I have got Office 2019 installed and now I'm getting the 0x80040155 error.
I tried to solve it according to Wiki pages but I noticed soon, that the version table contains the versions only up to Outlook 2016 (9.6). That's why I'm asking if OGCS is compatible with Outlook 2019...

Thank you in advance!

**Comment by phw198** [comment]
Yes OGCS should work fine with Outlook 2019, just the wiki needed updating. I haven't been able to find any Microsoft official documentation on the registry key for Outlook 2019, but a handful of sites ([1](https://docs.zivver.com/en/admin/officeplugin/references/cleanup-registry-outlook-installation.html), [2](https://help.salesforce.com/articleView?id=000333021&type=1&mode=1)) mention it should be `9.7`. However, [others](https://forums.ivanti.com/s/article/Library-not-registered-Exception-from-HRESULT-0x8002801D-TYPE-E-LIBNOTREGISTERED-in-Outlook-Log-Files?language=en_US) say it is still `9.6`, and since the release version stayed at `16` for Outlook 2016,2019,M365 I'm inclined to think it may well be `9.6`. 

FWIW, I have M365 Outlook and it's `9.6`.

Please could you confirm if your registry contains the below, or only `9.6`?
`HKEY_CLASSES_ROOT\TypeLib\{00062FFF-0000-0000-C000-000000000046}\9.7`

**Comment by schwaben-github** [comment]
My registry contains only
`HKEY_CLASSES_ROOT\TypeLib\{00062FFF-0000-0000-C000-000000000046}\9.6`
 
After registering the `Ole32.dll` I tried also to enter the value `9.7` under
`Computer\HKEY_CLASSES_ROOT\Interface\{00063001-0000-0000-C000-000000000046}\TypeLib` `Version` without success.

Should I try to rename the `HKEY_CLASSES_ROOT\TypeLib\{00062FFF-0000-0000-C000-000000000046}\9.6` into `\9.7`, together with the value of `Version` under the `Computer\HKEY_CLASSES_ROOT\Interface\{00063001-0000-0000-C000-000000000046}\TypeLib` changed to `9.7`, because obviously it doesn't work with original values?

**Comment by schwaben-github** [comment]
It's working now!
The path to `msoutl.olb` was slightly different. I just changed the `...\Root\...` to `...\root\...` under `HKEY_CLASSES_ROOT\TypeLib\{00062FFF-0000-0000-C000-000000000046}\9.6\0\Win64`. This is what changed compared to the old Office 2016 installation...

And I just found one error in Wiki:
Under the number "3. Ensure `MSOUTL.LIB` is Registered", there is following sentence:
"Execute the following command, where <msoutl.olb file> is the value taken from step No. 3:".
The step mentioned should be the *step number 1*.

Thanks for your help @phw198!
